src/amo/components/UserProfileEdit/index.js

```diff
@@ -154,11 +161,16 @@ export class UserProfileEditBase extends React.Component<Props, State> {
       });
     }
 
+    if (wasUpdating && !isUpdating) {
```

Comment:
This is done to scroll top in case of both error and success. In case of success while page is reloading user can see success notice.